### ISSUE TITLE
Validators table

### ISSUE THREAD

**Issue by dexturr** [vegaprotocol/frontend-monorepo#45]
<img width="948" alt="Screenshot 2022-04-13 at 19 49 11" src="https://user-images.githubusercontent.com/13255539/163249606-8e4e3027-da69-4618-b664-6d7cf4fdcefc.png">

- [ ] Remove existing list item (new table does not include your stake, this is intentional
- [ ] Table is sortable by users by clicking header
- [ ] Default sort == ranking score (high to low). use ag grid?
- [ ] Table creates horizontal scroll if/when needed
- [ ] Add h2 for "Nodes"
- [ ] Consider adding a checkbox to only show validating + Stand by validators (remove ones on the longer waiting list)

The intention of this table to list all the nodes who are known / active to the network, their overall status (validating etc) and the information needed to make informed decisions about who to delegate.

There is no single key value to show someone to help inform the staking decision, therefore lots are shown. ranking score best explains what is currently happening so has been chosen as default.

This is likely to be wider than the available area, this table layout is deemed a better default than a mobile first option.

## Cells and data
<img width="1129" alt="Screenshot 2022-04-13 at 19 57 17" src="https://user-images.githubusercontent.com/13255539/163250907-ea661deb-713f-4b0b-8293-d630c65dc6e4.png">

- Validator (avatar and name)
- Status (Shows either Consensus, Standby* or Candidate) See https://github.com/vegaprotocol/data-node/issues/509 not to be confuses with `node.status`
- Total stake this epoch `{ nodes{ stakedTotal}}`
- Share (use the total staked that epoch and the amount staked to show a share of the stake they have nominated (current epoch)
- Next epoch `{ nodes { pendingStake } }`
- Ranking score `See below`
- Staked score
- Performance
- Voting power

*this may be called ersatz in the API

<img width="205" alt="Screenshot 2022-04-13 at 19 55 50" src="https://user-images.githubusercontent.com/13255539/163250785-53a259b1-c4eb-49a3-9588-1eb98513eae2.png">

Future iterations may
- an indication return per delegated VEGA https://github.com/vegaprotocol/token-frontend/issues/1277
- Highlighting issues that prevent candidate becoming an ersatz/Standby 
- Show what version each node is running and what it intends to upgrade to
- Refine what columns are shown
- Consider alternatives to showing scores as floats
- Make it easier to see what scores are good/bad/or neither
- Consider whether to incudes normalised score

**Comment by campbellssource** [comment]
https://github.com/vegaprotocol/data-node/issues/509 could be done before this is started, but if it isn't we can just show the string.

**Comment by gordsport** [comment]
Hey @sam-keen 
I had a note on a duplicate issue that we needed:
 - https://github.com/vegaprotocol/data-node/issues/509
 
Is this the case or do we need a new issue to improve/update this feature once the above is merged?

**Comment by sam-keen** [comment]
@gordsport a new issue for when 509 is complete, thanks
